### DISCUSSION TITLE
Redmi Note 13 Pro Plus - NFC dont works

### DISCUSSION THREAD

**Discussion by fcourela** [NightscoutFoundation/xDrip#3512] [Q&A]
Anyone who has this model of mobile phone? since I bought it I can't use the Xdrip or Librelink application because the NFC sensor doesn't respond to the sensor reading. However, it's not an NFC problem because it works for payments. Can anyone help me or give me ideas for alternative workarounds?

**Comment by PhilTU** [comment]
There's more than one type of NFC/RFID protocol and frequency and the Libre may not be supported by that phone or chipset. This has arisen before, where someone could pair Bluetooth via RFID tag but not scan a Libre.

There can also be software limitations in the OS to prevent other apps gaining access to the payments system, this was why it took longer for Libre to be supported on iPhone as Apple wanted to reserve the system for Apple Pay. Do other users of this phone report issues with other types of RFID ? NFC tools is an app that would allow reading of things as a test https://play.google.com/store/apps/details?id=com.wakdev.wdnfc&hl=en_GB

**Reply by fcourela** [reply]
I've already tried this tool and read other tags to test it out. I was even able to read my own watch and somes cards without a problem. Libre just doesn't read.

**Reply by PhilTU** [reply]
Literally no reaction to the sensor?

**Reply by fcourela** [reply]
Nothing. The only time I was able to do this was by attaching another cell phone with NFC to the sensor, which seems to have forced it to read the sensor. Otherwise, it reads everything but the sensor.

**Reply by PhilTU** [reply]
This is the readour from NFC Tools on a Libre 2+ maybe you can ask if the specific protocol is supported ISO 15693 
![Screenshot_20240603-173346](https://github.com/NightscoutFoundation/xDrip/assets/76240903/aa7ea390-b77e-4c02-9732-198c5ba1140b)

**Reply by fcourela** [reply]
Do you have any idea of the most appropriate channel to ask this question?

**Reply by PhilTU** [reply]
Support from phone supplier.

How long since you bought it?

**Reply by fcourela** [reply]
I bought it about two months ago, but sending it back to the shop to activate the guarantee seems to me to be something that might be out of place, because the mobile phone works 100%. I've tried to find some kind of xiaomi support, but I haven't found a decent channel.

**Reply by PhilTU** [reply]
It's not working 100% if the NFC doesn't scan your sensor. These issues do arise with particular chipsets or software implementation, there's one with an EU version of a high end Samsung phone that affects the Libre 3.

Where are you based ? First line of approach might be to ask customer services if ISO 15693 RFID tags are supported at 13.56 MHz.

I was hoping you just got it and could make a 14 day return.

Others with same problem at https://www.reddit.com/r/Freestylelibre/comments/1aobhsh/redmi_note_13_pro_5g_amd_libre2_sesnors/
